conf/schema_example.conf

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If this file is used for integration tests, we should include it in the Docker image used for the test and not in the final binary tgz

Reply:
@merlimat - I am adding this to conf as same we adding an example yaml file for function spec. this serves as a template or an example conf for people to know how to create a schema.  that is the original consideration.

I used that directly for integration tests since it is available in the binary distribution.

Hope this clarify.